docs/docs/setup-vast/tune.md

```diff
@@ -0,0 +1,219 @@
+
+### Caching
+
+VAST groups table slices with the same schema in a *partition*. When building a
+partition, the parameter `max-partition-size` sets an upper bound on the number
+of records in a partition, across all table slices.
+
+A LRU cache of partitions accelerates queries to recently used partitions. The
+parameter `max-resident-partitions` controls the number of partitions in the LRU
+cache.
+
+### Catalog
+
+The catalog manages partition meta data and is responsible for deciding whether
+a partition qualifies for a certain query. It does so by maintaining *sketch*
+data structures (e.g., Bloom filters, summary statistics) for each partition.
+Sketches are highly space-efficient at the cost of being probabilistic and
+yielding false positives.
+
+Due to this characteristic sketches can grow sub-linear, doubling the number of
+events in a sketch does not lead to a doubling of the memory requirement.
+Because the catalog must be traversed in full for a given query it needs to be
+maintained in active memory to provide high responsiveness.
+
+As a consequence, the overall amount of data in a VAST instance and the
+`max-partition-size` determine the memory requirements of the catalog. The
+option `max-partition-size` is inversely linked to the number of sketches in the
+catalog. That means increasing the `max-partition-size` is an effective method
+to reduce the memory requirements for the catalog.
+
+### Tune sketch parameters
+
+A false positive can have substantial impact on the query latency by
+materializing irrelevant partitions, which involves unnecessary I/O. Based on
+the cost of I/O, this penalty may be substantial. Conversely, reducing the false
+positive rate increases the memory consumption, leading to a higher resident set
+size and larger RAM requirements.
+
+You can control this space-time trade-off in the configuration section
+`vast.index` by specifying index *rules*. Each rule corresponds to one sketch
+and consists of the following components:
+
+- `targets`: a list of extractors to describe the set of fields whose values to
+  add to the sketch.
+- `fp-rate`: an optional value to control the false-positive rate of the sketch.
+
+VAST does not create field-level sketches unless a dedicated rule with a
+matching target configuration exists.
+
+#### Example
+
+```yaml
+vast:
+  index:
+    rules:
+      - targets:
+          # field synopses: need to specify fully qualified field name
+          - suricata.http.http.url
+        fp-rate: 0.005
+      - targets:
+          - :addr
+        fp-rate: 0.1
+```
+
+This configuration includes two rules (= two sketches), where the first rule
+includes a field extractor and the second a type extractor. The first rule
+applies to a single field, `suricata.http.http.url`, and has false-positive rate
+of 0.5%. The second rule creates one sketch for all fields of type `addr` that
+has a false-positive rate of 10%.
+
+### Approximate memory requirements
```

Comment:
I find that this section distracts from tuning. It's much too complicated for most readers, so maybe it should go into some separate section with advanced tutorials or so. It also appears to be out of date already.